author.rb

```diff
@@ -0,0 +1,29 @@
+class Author
+  attr_accessor :first_name, :last_name, :id, :items
+
+  def initialize(first_name:, last_name:)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+    @id = Random.rand(1..1000)
+  end
```

Comment:
- [ ] Kindly, add the id field in the construction because you need to use it when retrive data from json file like:
     - def initialize(id: nil, irst_name:, last_name:)
            @id = id || Random.rand(1..1000)
             ........
      end

Reply:
Great review performed @Mo7amedSMahdi, let me fix the changes requested 👍